website/src/models/group.js

```diff
@@ -151,6 +151,50 @@ schema.statics.getGroup = async function getGroup (options = {}) {
   return group;
 };
 
+schema.statics.getGroups = async function getGroups (options = {}) {
+  let {user, types, groupFields = basicFields, sort = '-memberCount', populateLeader = false} = options;
+  let queries = [];
+
+  types.forEach(type => {
```

Comment:
Yea, this can probably be refactored, but I took the query from the getGroups route. I'm not sure who wrote it. But, it might need to be a separate request to update the function.